cmd/nodepool/azure/create.go

```diff
@@ -141,12 +144,12 @@ func NewCreateCommand(coreOpts *core.CreateNodePoolOptions) *cobra.Command {
 	return cmd
 }
 
-func (o *CompletedAzurePlatformCreateOptions) UpdateNodePool(_ context.Context, nodePool *hyperv1.NodePool, _ *hyperv1.HostedCluster, _ crclient.Client) error {
-	nodePool.Spec.Platform.Azure = o.NodePoolPlatform()
+func (o *CompletedAzurePlatformCreateOptions) UpdateNodePool(_ context.Context, nodePool *hyperv1.NodePool, hc *hyperv1.HostedCluster, _ crclient.Client) error {
```

Comment:
do we need the hc in this function?

Reply:
I thought I did and forgot to remove that.